src/expandable-block/block.scss

```diff
@@ -167,4 +167,35 @@
       margin-left: $iui-s;
     }
   }
+
+  &.iui-borderless {
+    @include themed {
+      background-color: transparent;
+      border: initial;
+    }
+
+    .iui-header {
+      @include themed {
+        background-color: transparent;
+        border: initial;
+      }
+
+      &:hover {
+        @include themed {
+          background-color: t(iui-color-background-4);
+        }
+      }
+    }
+
+    .iui-expandable-content {
+      @include themed {
+        background-color: transparent;
+        border: initial;
+      }
+
+      > div {
+        padding: 0;
```

Comment:
Are you sure about removing this padding?

Reply:
Isssue #454 asks that the padding be removed in the content area. I assumed it was because the padding was used to give space between the content and its border. Now that the border is removed, I don't see a need for the padding